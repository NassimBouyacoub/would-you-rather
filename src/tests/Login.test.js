import { render, screen } from "@testing-library/react";
import Login from '../Components/Login'
import React from 'react'
import { Provider } from "react-redux";
import {store} from '../index'

test("testing login component", () => {
    var component = render(
        <Provider store={store}>
            <Login />
        </Provider>
    )
    expect(component).toMatchSnapshot()
})