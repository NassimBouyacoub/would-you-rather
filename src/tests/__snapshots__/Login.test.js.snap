// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing login component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row"
      >
        <div
          class="col"
        >
          <nav
            class="navbar navbar-expand navbar-dark bg-dark"
          >
            <div
              class="container"
            >
              <span
                class="navbar-brand"
              >
                Would you rather?
              </span>
              <div
                class="me-auto navbar-nav"
              >
                <a
                  aria-current="page"
                  class="nav-link active"
                  href="/"
                >
                  Home
                </a>
                <a
                  class="nav-link"
                  href="/newQuestion"
                >
                  New Question
                </a>
                <a
                  class="nav-link"
                  href="/leaderboard"
                >
                  Leaderboard
                </a>
              </div>
              <div
                class="justify-content-end navbar-collapse collapse"
              >
                <h2>
                  none
                </h2>
              </div>
            </div>
          </nav>
        </div>
      </div>
      <br />
      <div
        class="row"
        style="text-align: center;"
      >
        <div
          class="col-md-4 offset-md-4"
        >
          <div
            class="card"
            style="text-align: center;"
          >
            <div
              class="card-body"
            >
              <img
                class="card-img"
                src="http://www.webstickersmuraux.com/fr/img/foma288-jpg/folder/products-listado-merchant/poster-xxl-bob-leponge.jpg"
              />
               
              <div
                class="dropdown"
              >
                <button
                  aria-expanded="false"
                  class="dropdown-toggle btn btn-success"
                  id="dropdown-basic"
                  type="button"
                >
                  Choose a user
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <br />
      <div
        class="row"
        style="text-align: center;"
      >
        <div
          class="col"
        >
          <p>
            Created By: 
            <a
              href="https://fr.linkedin.com/in/nassimbouyacoub"
            >
              Nassim
            </a>
          </p>
        </div>
      </div>
    </div>
    <div>
      <div
        class="card"
        style="text-align: center;"
      >
        <div
          class="card-body"
        >
          <img
            class="card-img"
            src="http://www.webstickersmuraux.com/fr/img/foma288-jpg/folder/products-listado-merchant/poster-xxl-bob-leponge.jpg"
          />
           
          <div
            class="dropdown"
          >
            <button
              aria-expanded="false"
              class="dropdown-toggle btn btn-success"
              id="dropdown-basic"
              type="button"
            >
              Choose a user
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="card"
      style="text-align: center;"
    >
      <div
        class="card-body"
      >
        <img
          class="card-img"
          src="http://www.webstickersmuraux.com/fr/img/foma288-jpg/folder/products-listado-merchant/poster-xxl-bob-leponge.jpg"
        />
         
        <div
          class="dropdown"
        >
          <button
            aria-expanded="false"
            class="dropdown-toggle btn btn-success"
            id="dropdown-basic"
            type="button"
          >
            Choose a user
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
